packages/ui/core-components/src/lib/unsorted/viz/map/components/Legend Components/LegendIcons.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+	/** @type {boolean} */
+	export let hideLegend = false;
+	export let handleLegendClick;
+
+	/** @type {string | undefined} */
+	export let legendType = undefined;
+	/** @type {'left' | 'right' | 'bottom' | 'top'} */
+	export let direction = 'bottom';
```

Comment:
`bottom` and `top` aren't directions, they should be `down` and `up`, respectively